chartjs-chart-sankey.esm.d.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  CartesianScaleTypeRegistry,
+  Chart,
+  ChartComponent,
+  DatasetController,
+  SankeyParsedData, ScriptableContext,
+} from "chart.js";
+import {Element} from "chart.js/types/element";
```

Comment:
Shouldn't the Element be in the upper list too?

Reply:
The auto-import feature added it this way. (which is strange)